lib/src/google_navigation_flutter_platform_interface.dart

```diff
@@ -532,6 +532,9 @@ abstract mixin class MapViewAPIInterface {
   /// Register camera changed listeners.
   Future<void> registerOnCameraChangedListener({required int viewId});
 
+  // Sets the map padding for the map view.
+  Future<void> setPadding({required int viewId, required MapPadding padding});
```

Comment:
I would give EdgeInsets instead of new MapPadding class.
https://api.flutter.dev/flutter/painting/EdgeInsets-class.html